modules/thermo-cycler/thermo-cycler-arduino/gcode.cpp

```diff
@@ -211,7 +217,7 @@ void GcodeHandler::add_debug_response(String param, float val)
 {
   Serial.print(param);
   Serial.print(F(": "));
-  Serial.print(val);
+  Serial.print(val, 4);
```

Comment:
Is there a reason that this isn't `SERIAL_DIGITS_IN_RESPONSE` (which I believe equals 4), instead of this magic number ?